support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx

```diff
@@ -211,19 +231,94 @@ export function PaymentTsAndCs({
 		<div css={container}>
 			<FinePrint mobileTheme={mobileTheme}>
 				{inTier3 && (
-					<ThreeTierTerms
+					<TierThreeTerms
 						paymentFrequency={contributionType === 'ANNUAL' ? 'year' : 'month'}
 					/>
 				)}
 				{inSupporterPlus &&
-					copyAboveThreshold(
-						contributionType,
-						productNameAboveThreshold,
-						'SupporterPlus',
-						promotion,
-					)}
-				{inSupport && copyBelowThreshold(contributionType)}
+					copyAboveThreshold(contributionType, productKey, promotion)}
+				{inSupport && copyBelowThreshold(contributionType, countryGroupId)}
 			</FinePrint>
 		</div>
 	);
 }
+
+export function SummaryTsAndCs({
+	contributionType,
+	countryGroupId,
+	currency,
+	amount,
+	amountIsAboveThreshold,
+	productKey,
+	cssOverrides,
+}: PaymentTsAndCsProps): JSX.Element {
+	const inTier3 = productKey === 'TierThree' && amountIsAboveThreshold;
```

Comment:
Why do we need `amountIsAboveThreshold` if the amount is defined by the product API

Reply:
Agree not needed for Summary component, removed